lerna.json

```diff
@@ -2,5 +2,7 @@
   "packages": [
     "packages/*"
   ],
+  "npmClient": "yarn",
+  "useWorkspaces": true,
```

Comment:
Will this fix the issue with running `npm install`?

Reply:
Unfortunately doesn't fully fix it, but is part of my work to get there.  It's at least setting this mono repo up in a more explicit way.

Reply:
I need to do some more research on handling workspaces in `npm` and generally setting up a monorepo to work with both `npm` and `yarn.